parsons/shopify/shopify.py

```diff
@@ -0,0 +1,96 @@
+from datetime import date, datetime, timedelta
+from enum import Enum
+from io import StringIO
+import requests
+
+from parsons.etl.table import Table
+from parsons.utilities import check_env
+
+class QueryTypes(Enum):
+    COUNT = "count"
+    ORDERS = "orders"
+
+class Shopify(object):
+    """
+    Instantiate the Shopify class
+
+    `Args:`
+        subdomain: str
+            The Shopify subdomain (e.g. ``myorg`` for myorg.myshopify.com) Not required if
+            ``SHOPIFY_SUBDOMAIN`` env variable set.
+        password: str
+            The Shopify account password. Not required if ``SHOPIFY_PASSWORD`` env
+            variable set.
+        api_key: str
+            The authorized ActionKit user password. Not required if ``SHOPIFY_API_KEY``
+            env variable set.
+    """
+
+    def __init__(self, subdomain=None, password=None, api_key=None):
+        self.COUNT = QueryTypes.COUNT
+        self.ORDERS = QueryTypes.ORDERS
+        self.subdomain = check_env.check('SHOPIFY_SUBDOMAIN', subdomain)
+        self.password = check_env.check('SHOPIFY_PASSWORD', password)
+        self.api_key = check_env.check('SHOPIFY_API_KEY', api_key)
+
+    def get_csv(self, tbl=None):
+        """
+        Generate StringIO from a Table
+        """
+        csv_path = tbl.to_csv()
+        csv_contents = ''
+        with open(csv_path, newline='') as csv_file:
+            csv_contents = csv_file.read()
+        
+        return StringIO(csv_contents).getvalue()
+
+    def get_orders_tbl(self, url=None):
```

Comment:
probably rename this to `get_orders` -- Also document the arguments -- i might suggest accepting the arguments in get_query() -- i.e. duplicate them and then inside call get_query yourself.

Please document the public methods that you expect a user of this class to use just like you did for init.  Methods that you don't expect a public user to use, should start with `_`.